src/vstt/vis.py

```diff
@@ -150,25 +150,34 @@ def _make_stats_txt(display_options: DisplayOptions, stats: pd.Series) -> str:
                 if (
                     stat == "to_target_success" or stat == "to_center_success"
                 ) and "to_target_success_trial" not in stats:
-                    if not display_options["averages"]:
-                        stats[stat] = True if stats[stat] == 1 else False
-                        txt_stats += f"{label} (to {destination}): {stats[stat]}\n"
-                    else:
-                        txt_stats += f"{label} (to {destination}): {stats[stat]:.3%}\n"
+                    stats[stat] = True if stats[stat] == 1 else False
+                    txt_stats += f"{label} (to {destination}): {stats[stat]}\n"
                 elif (
                     stat == "to_target_success" or stat == "to_center_success"
                 ) and "to_target_success_trial" in stats:
-                    if not display_options["averages"]:
-                        txt_stats += f"{label} (to {destination}): {stats[stat]:.3%}\n"
-                    else:
-                        txt_stats += f"{label} (to {destination}) in block: {stats[stat+'_trial']:.3%}\n"
+                    txt_stats += f"{label} (to {destination}): {stats[stat]:.3%}\n"
```

Comment:
Zero decimal places is probably enough for the percentages, i.e. `.0%` instead of `.3%`